src/WebAssert.php

```diff
@@ -335,10 +334,9 @@ public function responseContains($text)
     public function responseNotContains($text)
     {
         $actual = $this->session->getPage()->getContent();
-        $regex = '/'.preg_quote($text, '/').'/ui';
         $message = sprintf('The string "%s" appears in the HTML response of this page, but it should not.', $text);
 
-        $this->assert(!preg_match($regex, $actual), $message);
+        $this->assert(strpos($actual, $text) === false, $message);
```

Comment:
The `i` modifier of `preg_match` was to match in case-insensitive manner, so to replicate that you need to use `stripos` instead of `strpos`.